services/management/ia/templates_service.go

```diff
@@ -151,6 +162,126 @@ func (svc *TemplatesService) loadFile(ctx context.Context, file string) ([]saas.
 	return rules, nil
 }
 
+type ruleFile struct {
+	Group []ruleGroup `yaml:"groups"`
+}
+
+type ruleGroup struct {
+	Name  string `yaml:"name"`
+	Rules []rule `yaml:"rules"`
+}
+
+type rule struct {
```

Comment:
Why not use `alert.Rule`?

Reply:
but that type represents a template rule and what we are using is an alert rule, they have different fields so can't be reused here.

Reply:
Probably we need to rename `alert.Rule` into `alert.Template`  😑